client/package.json

```diff
@@ -18,6 +18,7 @@
     "leaflet": "^1.7.1",
     "node-sass": "^4.14.1",
     "phaser": "^3.54.0",
+    "prop-types": "^15.7.2",
```

Comment:
Humm... weird that this is missing. I must have been using a globally installed package or something.